ddl/ddl_api.go

```diff
@@ -394,6 +394,12 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef, o
 		col.Flag &= ^mysql.BinaryFlag
 		col.Flag |= mysql.ZerofillFlag
 	}
+	// If you specify ZEROFILL for a numeric column, MySQL automatically adds the UNSIGNED attribute to the column.
+	// See https://dev.mysql.com/doc/refman/8.0/en/numeric-type-overview.html for more details.
+	// But some types like bit and year, won't show its unsigned flag in `show create table`.
```

Comment:
we should not support https://github.com/pingcap/tidb/issues/6688?

Reply:
It seems it still could work in some client/driver, see https://github.com/pingcap/tidb/pull/7525

Reply:
myabe we need use `5.7`